pkgs/games/minecraft-servers/derivation.nix

```diff
@@ -16,7 +16,7 @@ stdenv.mkDerivation {
 
     makeWrapper ${lib.getExe jre_headless} $out/bin/minecraft-server \
       --append-flags "-jar $out/lib/minecraft/server.jar nogui" \
```

Comment:
Will the \ not cause issues when there is nothing below?

Reply:
I think it is fine as long as there is a new line below

```
❯ bat example
───────┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
       │ File: example
───────┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
   1   │ echo \
   2   │   true \
   3   │   true \
   4   │
   5   │   true
───────┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

❯ ./example
true true
```

Reply:
Feel free to force push a better string concatenation/interpolation.